spec/pQueue.js

```diff
@@ -0,0 +1,84 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import pQueue from '../src/pQueue'
+
+chai.use(chaiChange)
+
+describe('pQueue', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(pQueue).to.be.a('function')
+  })
+
+  context('enqueue()', () => {
+    it('adds an element with priority to the back of the queue.', () => {
+      const aQueue = new pQueue()
+
+      aQueue.enqueue('dude', 90)
+      aQueue.enqueue('cool', 20)
+      aQueue.enqueue('turd', 50)
+      expect(aQueue.enqueue('foo',70)).to.eql([{name: 'dude', priority: 90}, {name: 'cool',  priority: 20}, {name: 'turd', priority: 50}, {name: 'foo', priority: 70}])
```

Comment:
Consider formatting this line to make it more readable (it scrolls off the screen in github if greater than 80 characters):
```js
expect(aQueue.enqueue( 'foo',70 )).to.eql([
  {name: 'dude', priority: 90}, 
  {name: 'cool',  priority: 20}, 
  {name: 'turd', priority: 50}, 
  {name: 'foo', priority: 70}
])
```
This makes it a little for a developer outside of your project to read and understand your code.

Looking at this test, I think there is a flaw with your `enqueue` implementation - enqueue should not return anything, and you are testing the return value of `enqueue`.  Consider the following approach that leverages the API of the `PriorityQueue` object in order to perform the test:
```js
const queue = new pQueue()

queue.enqueue( 'highest', 90 )
queue.enqueue( 'lowest', 20 )
queue.enqueue( 'medium', 50 )
queue.enqueue( 'high medium', 70 )

expect( queue.front() ).to.equal( 'highest' )
```

Finally, consider renaming the identifier for your `PriorityQueue` class from `pQueue` to `PriorityQueue` - it is standard practice to use capital camel case for type identifiers, and to use lower camel case for variable identifiers.